web/src/components/button.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Link } from 'gatsby';
+
+export default function Button({
+    isPrimary,
+    isPrimaryDark,
+    isSecondary,
+    isSecondaryDark,
+}) {
+    const primaryStyle =
+        'bg-black text-white hover:bg-brand-1 visited:bg-gray-300';
+    const primaryDarkStyle =
+        'bg-white text-black hover:bg-gray-300 visited:bg-brand-1';
+    const secondaryStyle =
+        'bg-white text-black hover:bg-brand-1 hover:text-brand-1 visited:text-gray-300';
```

Comment:
I don't think we want to change the color on `:visited` links. You should add `:active` styles to match `:hover` styles for keyboard users.

Reply:
@asasmith I am not sure I am following, the style was for a visited link?

Reply:
@kcbreuil I don't think we want to change style for `:visited` links. The only states we should be concerned with are `:hover`, `:focus`, and maybe `:active`. 

`:visited` applies styles to links you've visited. I'm not sure how the browser determines that (my guess is caching or local storage). [mdn docs](https://developer.mozilla.org/en-US/docs/Web/CSS/:visited)

Reply:
@asasmith is deactivated active? 
<img width="435" alt="Screen Shot 2021-05-27 at 5 03 57 PM" src="https://user-images.githubusercontent.com/61708936/119896800-8910ed00-bf0d-11eb-89eb-6f5636b0a8aa.png">

Reply:
No, that would be for an actual button that is an inactive state (like for a form that isn't filled out).

Reply:
Okay then I think we need to get clarity from Josh cause he did not give us designs for the other states youre suggesting @asa

Reply:
Okay then I think we need to get clarity from Josh cause he did not give us designs for the other states youre suggesting @asa

Reply:
@kcbreuil I think it's safe to assume that hover and focus states should behave the same. Hover is a mouse operation focus is for keyboard navigation but they both communicate the same thing to the user.